docs/operate/validators/monitoring-alerting.mdx

```diff
@@ -15,12 +15,31 @@ If running as a Docker image, make sure to port-forward the metrics endpoint por
+| `Diff observed - processed checkpoints` | The difference between checkpoints which validator observed on blockchain and the checkpoints which validator signed and published. This chart should be zero most of the time with occasional and short lived positive values. If it is positive for extended period of time, it means that validator has issues with signing and publishing checkpoints.                |
+| `Latest Block Diffs (30m)`              | The block height of the RPC node the agent is connected to, visualized as 30 min increases ("Latest block diffs"). If this metric is not increasing, the RPC may be unhealthy and need changing.                                                                                                                                                                          |
+| `Contract Sync Liveness`                | Renders metric `hyperlane_contract_sync_liveness` as it is. The graph should always go up. If it is constant, it indicate an issue and it should be investigated.                                                                                                                                                                                                         |
+| `Contract Sync Block Diffs (30m)`       | Renders metric `hyperlane_contract_sync_block_height` as 30 min increases. It should approximately match the number of blocks produced by the chain during last 30 minutes. If the metric deviate significantly in any direction, becomes zero, it should be investigated.                                                                                                |
+| `Cursor Block Diffs (30m)`              | Renders metric `hyperlane_cursor_current_block` as 30 min increases. It should approximately match the number of blocks produced by the chain during last 30 minutes. If the metric deviate significantly in any direction, becomes zero, it should be investigated.                                                                                                      |
+| `Error log count diffs (30m)`           | The total number of errors logged, visualized as 30 min increases ("Error log count diffs"). If the derivative of this metric exceeds 1 over the last hour, at least a low-severity alert is warranted. Note that the dashboard query groups metrics by kubernetes pod name, so you may need to adjust this query if you are not running in a kubernetes environment.     |
+| `Warning log count diffs (30m)`         | The total number of warnings logged, visualized as 30 min increases ("Warning log count diffs"). If the derivative of this metric exceeds 1 over the last hour, at least a low-severity alert is warranted. Note that the dashboard query groups metrics by kubernetes pod name, so you may need to adjust this query if you are not running in a kubernetes environment. |
```

Comment:
bummer about this pod thing, though I get that there's no alternative. Wonder if this is something we should have better support for, but not a priority